lib/ansible/modules/net_tools/ldap/ldap_entry.py

```diff
@@ -145,7 +145,7 @@ def _load_attrs(self):
                 attrs[name] = []
 
             if isinstance(value, list):
-                attrs[name] = [map(to_bytes, value)]
+                attrs[name] = list(map(to_bytes, value))
```

Comment:
Note: better to do this like:
``` python
attrs[name] = [to_bytes(v) for v in value]
```
* map() is slower than a list comprehension (and map + list() would be even slower)
* list comprehensions are more pythonic than map() or filter()